appveyor.yml

```diff
@@ -1,34 +1,39 @@
-version: 5.0.1.{build}
-
-configuration: Release
-
-os: Visual Studio 2019
-
-environment:
-  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
-  NUGET_XMLDOC_MODE: skip
-
-skip_tags: true
-
-nuget:
-  disable_publish_on_pr: true
-
-clone_depth: 1
-
-test: off
-
-build_script:
-- ps: .\build.ps1
-
-artifacts:
-- path: artifacts\packages\**\*.nupkg
-  name: MyGet
-
-deploy:
-- provider: NuGet
-  server: https://www.myget.org/F/autofac/api/v2/package
-  api_key:
-    secure: rCUEY75fXN0wxtMy6QL4jCrLdaYbxIBzIXWeN+wEu/XDpyqimzreOc5AH5jMd5ah
-  skip_symbols: true
-  symbol_server: https://www.myget.org/F/autofac/symbols/api/v2/package
-  artifact: MyGet
+image: Ubuntu
+
+version: '6.0.0.{build}'

```

Comment:
I don't think we're ready to go to 6.0 yet, are we?

Reply:
We dropped support for `net461` which is breaking change. Wouldn't the version raise accurate by the definition of semantic-versioning?

Reply:
Oh, yeah, I see it now. It happened in a different pull request so I missed it. You're right, 6 it is.